include/deal.II/tests/tests.h

```diff
@@ -0,0 +1,149 @@
+//-----------------------------------------------------------
+//
+//    Copyright (C) 2021 by the deal.II authors
+//
+//    This file is part of the deal.II library.
+//
+//    The deal.II library is free software; you can use it, redistribute
+//    it, and/or modify it under the terms of the GNU Lesser General
+//    Public License as published by the Free Software Foundation; either
+//    version 2.1 of the License, or (at your option) any later version.
+//    The full text of the license can be found in the file LICENSE.md at
+//    the top level directory of deal.II.
+//
+//---------------------------------------------------------------
+
+#ifndef dealii_tests_h
+#define dealii_tests_h
+
+#include <deal.II/base/logstream.h>
+#include <deal.II/base/mpi.h>
+
+
+DEAL_II_NAMESPACE_OPEN
+
+// Function to initialize deallog. Normally, it should be called at
+// the beginning of main() like
+//
+// initlog();
+//
+// This will open the correct output file, divert log output there and
+// switch off screen output. If screen output is desired, provide the
+// optional first argument as 'true'.
+std::string   deallogname;
+std::ofstream deallogfile;
```

Comment:
This is putting things into the global `dealii` namespace that I don't think should be there. If you must, we could have a nested namespace `dealii::Testing` where all of this could go.